src/scheduler/parse.c

```diff
@@ -934,7 +943,6 @@ init_config()
 	conf.update_comments = 1;
 
 	conf.decay_time = 86400;	/* default decay time period is 24 hours */
-	conf.sync_time = 86400;
```

Comment:
was this intentional?

Reply:
Oh, yeah, it was.  I can back it out if you want.  That's literally the only place sync time is set/used.

Reply:
oh, so should we just remove conf.sync_time altogether?

Reply:
Sure, let me remove it.  I should probably file a doc bug to get it removed from the man page because you can't set it.